app/views/layouts/application.html.erb

```diff
@@ -50,6 +50,9 @@
     <% if @node.main_image %>
         <meta property="og:image" content="https://<%= request.host %><%= @node.main_image.path(:default) %>" />
         <meta name="twitter:image" content="https://<%= request.host %><%= @node.main_image.path(:default) %>">
+        <% elsif @node.images %>
```

Comment:
Aha so this is why my trick isn't working. If an external image is pasted in, it won't show up in `@node.images`. And, `@node.scraped_image` won't appear here either -- in fact, an image that could have been uploaded and then removed (say with private info) could mistakenly be used if we go with this logic. I'm going to suggest we try swapping all `@node.images` for `@node.scraped_image` here!

```suggestion
        <% elsif scraped_image = @node.scraped_image %>
```